packages/console/src/components/Status/index.tsx

```diff
@@ -0,0 +1,18 @@
+import classNames from 'classnames';
+import React, { ReactNode } from 'react';
+
+import styles from './index.module.scss';
+
+type Props = {
+  status: 'operational' | 'offline';
```

Comment:
enabled / disabled?

Reply:
we may have other status like: warning, pending...

Reply:
I think they are not conflicted. `enabled / disabled / pending / warning` still looks fine to me. The main concern is the `operational`, sounds weird to me. Not sure about the others.

Reply:
created LOG-1856 to discuss.